salt/cache/mysql_cache.py

```diff
@@ -90,7 +88,7 @@ def __virtual__():
     return bool(MySQLdb), "No python mysql client installed." if MySQLdb is None else ""
 
 
-def run_query(conn, query, retries=3):
+def run_query(conn, query, args=None, retries=3):
```

Comment:
can we add `args=None` to the end in case anyone is calling the `retries` kwarg as positional parameters?